edge/client/portainer_edge_async_client.go

```diff
@@ -415,14 +415,18 @@ func (client *PortainerAsyncClient) executeAsyncRequest(payload AsyncRequest, po
 	defer resp.Body.Close()
 
 	if resp.StatusCode != http.StatusOK {
-		errorData := parseError(resp)
-		logError(resp, errorData)
-
-		if errorData != nil {
-			return nil, errors.New(errorData.Message + ": " + errorData.Details)
+		ctxMsg := "AsyncEdgeAgentExecuteAsyncRequest"
+		errMsg := "AsyncEdgeAgent failed to execute async request"
+		if err := decodeNonOkayResponse(resp, ctxMsg); err != nil {
+			log.
+				Error().Err(err.Err).
+				Str("context", ctxMsg).
+				Str("response message", err.Message).
```

Comment:
```suggestion
				Str("response_message", err.Message).
```
We need to have a consistent naming for variables, I suggest that we use snake case for this. I will update the guidelines with that.